### PR TITLE
Update Bulk Data section with addition of email_opt_in_types

### PR DESCRIPTION
**Pull request by lavaturtle**
This updates the Bulk Data documentation section. It adds the new `email_opt_in_types` table, and updates the documentation for `signatures`, `attendees`, and `users` to reflect the addition of `email_opt_in_type_id`.

(The new headers were pulled from a recent export on staging; AFAIK they reflect what we are currently exporting. The new schemas were copied from `db/structure.sql`, with the removal of columns that are excluded from the bulk exports.)

### CODE REVIEW

File: source/data/email_opt_in_types_schema.sql [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+SET statement_timeout = 0;
+SET client_encoding = 'UTF8';
+
+SET search_path = public, pg_catalog;
+
+CREATE TABLE email_opt_in_types (
+    id BIGSERIAL PRIMARY KEY,
+    organisation_id bigint NOT NULL,
+    kind character varying NOT NULL,
+    name character varying NOT NULL,
```
Comment (anero) [inline source/data/email_opt_in_types_schema.sql]: Could you remove this column that we're dropping soon

File: source/data/signatures_schema.sql [hunk 2]
```diff
@@ -49,5 +49,6 @@ CREATE TABLE signatures (
     from_one_click boolean,
     eu_data_processing_consent boolean,
     consent_content_version_id bigint,
-    daisy_chain_id_used bigint
-);
\ No newline at end of file
+    daisy_chain_id_used bigint,
```
Comment (anero) [inline source/data/signatures_schema.sql:52]: This is definitely out of scope for this PR, but we should make sure to only export FKs that refer to other tables that we also export

**Review by anero — APPROVED** [review]
:+1:

File: source/data/signatures_schema.sql (hunk 2 above)
Comment (lavaturtle) [reply source/data/signatures_schema.sql:52]: Yeah... I feel like this update has revealed several columns we're exporting and probably shouldn't. Like attendee token.

**Pull request merged**